README

```diff
@@ -151,6 +151,32 @@ To create and play with a Test object from the example package,
 	}
 
 
+Parameter interpretation
+========================
+
+To pass extra parameters to the plugin, use a comma-separated
+parameter list separated from the output directory by a colon:
+
+
+	protoc --go_out=plugins=grpc,import_path=mypackage:. *.proto
+
+
+- `import_prefix=xxx` - a prefix that is prepended onto the beginning
+  of all imports. Useful for things like generating protos in a
+  subdirectory, or regenerating vendored protobufs in-place.
+- `import_path=foo/bar` - struct comments say, “Go import path of the
+  package we're generating code for” - this is ignored unless there is
+  no explicit `go_package` declared in any of the files. In that case,
+  this `import_path` is used as the package (ignoring any part of it
+  up to the rightmost slash).
+- `plugins=plugin1+plugin2` - specifies the list of sub-plugins to
+  load. The only plugin is `grpc` unless you've forked the `main`
+  package.
+- `Mfoo/bar.proto=quux/shme` - an explicit import mapping: this
```

Comment:
... - declares that `foo/bar.proto` is associated with Go package `quux/shme`.
  This is subject to the `import_prefix` parameter.